xml/admin_ceph_cephfs.xml

```diff
@@ -601,7 +601,14 @@ setfattr -n ceph.dir.pin -v 0 a/b # "a/b" is now pinned to rank 0
 </screen>
    <para>
     After you enable snapshots, all directories in the &cephfs; will have a
-    special <filename>.snap</filename> subdirectory.
+    special <filename>.snap</filename> subdirectory. Note that this is a virtual subdirectory.
+    In other words this directory does not appear in the directory listing of the parent directory,
+    but the name <filename>.snap</filename> can not be used as a file or directory name.
+    To access the <filename>.snap</filename> directory one has to explicitly access it, for exmaple
+<screen>
+&prompt.user;ls -la /<replaceable>CEPHFS_MOUNT</replaceable>/.snap/
+ </screen>
```

Comment:
```suggestion
<screen>
&prompt.user;ls -la /<replaceable>CEPHFS_MOUNT</replaceable>/.snap/
 </screen>
    </note>

Reply:
It is also failing the build @jan--f because you have a `<screen>`  tag inside the `<para>` If you move the `<screen>` outside, it will build just fine :)